File: CS#/Cylinder.cs
using System;

class Cylinder
{
    private double radius;
    private double length;

    public double Radius
    {
        get { return radius; }
        private set { radius = value; }
    }

    public double Length
    {
        get { return length; }
        private set { length = value; }
    }

    public Cylinder(double radius, double length)
    {
        Radius = radius;
        Length = length;
    }

    public double area()
    {
        return 2 * Math.PI * radius * radius;
    }

    public double volume()
    {
        return Math.PI * radius * radius * length;
    }

    public void printInfo()
    {
        Console.WriteLine("Cylinder");
        Console.WriteLine("Radius = " + radius);
        Console.WriteLine("Length = " + length);
        Console.WriteLine("Area = " + area());
        Console.WriteLine("Volume = " + volume());
    }
}

File: Python/Cylinder.py
import math

class Cylinder:
    # - radius : Real
    # - length : Real
    def __init__(self, radius, length):
        self.__radius = radius
        self.__length = length

    @property
    def radius(self):
        return self.__radius

    @property
    def length(self):
        return self.__length

    # + area() : Real
    def area(self):
        return 2 * math.pi * self.__radius * self.__radius

    # + volume() : Real
    def volume(self):
        return math.pi * self.__radius * self.__radius * self.__length

    # + printInfo()
    def printInfo(self):
        print("Cylinder")
        print("Radius =", self.__radius)
        print("Length =", self.__length)
        print("Area =", self.area())
        print("Volume =", self.volume())
